content/en/blog/_posts/2023-07-18-swap-beta1-graduation/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: blog
+title: "Kubernetes 1.28: NodeSwap graduates to Beta"
+date: 2023-07-18
+slug: swap-beta1-1.28-2023
+---
+
+**Author:** Itamar Holder (Red Hat)
+
+The 1.22 release introduced Alpha support for configuring swap memory usage for
+Kubernetes workloads on a per-node basis. Now, in release 1.28, swap configuration
+graduates to Beta1 with many new improvements.
```

Comment:
`Beta1` is not clear to a new user. We may say that Beta1 means that the FG is Beta but will not be enabled by default.

Reply:
Yup, you're absolutely right.
On second thought, let's not complicate the user with such internal details, I've simply changed it to `Beta`.